pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -658,9 +661,34 @@ private boolean isFilterAlwaysTrue(PinotQuery pinotQuery) {
   }
 
   private void logBrokerResponse(long requestId, String query, RequestContext requestContext, String tableName,
-      int numUnavailableSegments, ServerStats serverStats, BrokerResponseNative brokerResponse, long totalTimeMs) {
+      int numUnavailableSegments, ServerStats serverStats, BrokerResponseNative brokerResponse, long totalTimeMs,
+      @Nullable RequesterIdentity requesterIdentity) {
     LOGGER.debug("Broker Response: {}", brokerResponse);
 
+    boolean enableClientIpLogging = _config.getProperty(Broker.CONFIG_OF_BROKER_REQUEST_CLIENT_IP_LOGGING,
+        Broker.DEFAULT_BROKER_REQUEST_CLIENT_IP_LOGGING);
+    String clientIp = "";
```

Comment:
```suggestion
    String clientIp = "0";
```
I suggest not leaving it as null, so the parsing scripts will still work fine. many expect key=value pair. thanks.

Reply:
How about "unknown" which is more clear?

Reply:
I was thinking to minimize the line length

Reply:
Understood. In regular case the ip should be longer than 7 characters, so I feel it is worth spending 6 extra characters and make the log more clear. User might be confused when seeing an ip as `0`

Reply:
Sure, I can settle for UNKNOWN

Reply:
Or, "XXX" or just "X"?
I don't know, pick one. I think the line is getting a bit too long... but sure.

Reply:
Is the community decided on which one to pick : 0, unknown, XXX, X ?  I can make the change

Reply:
@abhs50 Let's do `unknown` for clarity since the line is already very long.
FYI #9122 is also touching this part, so if that one gets merged you might need to solve the conflict

Reply:
Updated. 
![image](https://user-images.githubusercontent.com/518685/181849296-9ba13214-c582-4c46-ab9c-b64d2a2e23f9.png)

Reply:
@Jackie-Jiang : Are we good to go on this one ? thanks